src/model/GroundHeatExchangerHorizontalTrench.cpp

```diff
@@ -259,6 +261,22 @@ namespace model {
       return false;
     }
 
+    ModelObject GroundHeatExchangerHorizontalTrench_Impl::clone(Model model) const {
+      auto groundHeatExchangerClone = StraightComponent_Impl::clone(model).cast<GroundHeatExchangerHorizontalTrench>();
+
+      groundHeatExchangerClone.setUndisturbedGroundTemperatureModel(undisturbedGroundTemperatureModel().clone(model));
+
+      return std::move(groundHeatExchangerClone);
+    }
+
+    std::vector<ModelObject> GroundHeatExchangerHorizontalTrench_Impl::children() const {
+      std::vector<ModelObject> result;
+
+      result.push_back(undisturbedGroundTemperatureModel());
+
+      return result;
+    }
```

Comment:
It this new `children` method necessary? I'm still not really sure how this method gets used downstream.